RO_UserRepositoryTest

```diff
@@ -0,0 +1,32 @@
+namespace TestProject2
+{
+    using System;
```

Comment:
move out usings from namespace(it's bad practice)

Reply:
So, is StyleCop rule about moving usings inside namespace is just bad?

Reply:
Actually it's not bad(it doesn't matter where are usings), I'd recommend changing it for outside as it's common practice)